### PR TITLE
No need for lodash and tmpPath

### PR DESCRIPTION
**Pull request by saif-o99**

### CODE REVIEW

File: src/CacheManager.ts [hunk 1]
```diff
@@ -102,7 +100,7 @@ export class CacheEntry {
       try {
         const result = await retry(() =>
           FileSystem.fetch(source, {
-            path: tmpPath,
+            path: path,
```
Comment (efstathiosntonas) [inline src/CacheManager.ts:103]: @saif-o99 this should be just `path` not `path: path`

**Comment by saif-o99** [comment]
Hey @efstathiosntonas, Could you please revisit the PRs once you get some free time?

**Comment by efstathiosntonas** [comment]
Hi @saif-o99, sorry for the delay, I’m in the middle of a major upgrade of an app and didn’t found some time to review. Will do it tomorrow

**Comment by saif-o99** [comment]
No worries at all

**Comment by efstathiosntonas** [comment]
Thanks for the contribution @saif-o99 !

**Comment by saif-o99** [comment]
You're welcome @efstathiosntonas 👍

**Comment by efstathiosntonas** [comment]
released on [3.0.0](https://github.com/georstat/react-native-image-cache/releases/tag/v3.0.0)

**Pull request merged**